nix/sources.json

```diff
@@ -1,14 +1,14 @@
 {
     "cardano-node": {
-        "branch": "master",
+        "branch": "1.13.0",
```

Comment:
Also, we shouldn't forget to update the readme matrix!

Reply:
Thanks, fixed.